src/core/system.c

```diff
@@ -1,6 +1,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <time.h>
```

Comment:
Windows build is working with no issues without that header. Has anyone been able to check if Mac works? @cochrane